ui/components/component-library/icon/icon.stories.tsx

```diff
@@ -60,51 +43,28 @@ export default {
   },
   argTypes: {
     name: {
-      control: 'select',
```

Comment:
Turns out you don't need to add select if options exist it's inferred https://storybook.js.org/docs/react/essentials/controls
<img width="977" alt="Screenshot 2023-03-24 at 7 11 00 PM" src="https://user-images.githubusercontent.com/8112138/227680825-3575e539-bb9c-4da6-a968-10b4a9d40494.png">